### ISSUE TITLE
Accessibility - VoiceOver does not read deleted character on macOS

### ISSUE THREAD

**Issue by adincebic** [microsoft/vscode#55042]
- VSCode Version: 1.25.1
- OS Version: macOS High Siera 10.13.6

Steps to Reproduce:

1. Turn voiceover on (command + F5)
2. Launch the VSCode
3. Create a file
4. Get the code editor in focus by pressing command + shift + E
5. Interact with the text by pressing option + ctrl + down arrow
6. Turn quick navigation off by pressing left and right arrows at the same time (you will hear "quick nav off")
7. Now, type some code or any kind of content and than try deleting character by character using backspace

Expected result: Voiceover should read the deleted character unless that feature is turned off in voiceover settings.

Actual result: Voiceover reads nothing, it basically stays quiet.

Does this issue occur when all extensions are disabled?: Yes/No
Yes

**Comment by isidorn** [comment]
This works with latest electrno-4-0. And works nicely on the web
Thus closing
@adincebic if you can still reproduce the issue with latest vscode insiders let us know and I will reopen https://code.visualstudio.com/insiders/
